guides/v2.0/javascript-dev-guide/javascript/custom_js.md

```diff
@@ -194,8 +194,8 @@ For information about how to initialize your custom JS component in a `.phtml` t
 To disable the auto-loading of default Magento JS components and widget initialization:
 <ol>
   <li>Create a <code>requirejs-config.js</code> file with the
-  following content: {% highlight JavaScript %} var config = {
-  deps: [ ] }; {% endhighlight %}
+  following content: ```JavaScript var config = {
```

Comment:
Would this just be single back ticks instead of three, since its inline?

Reply:
yes